torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -918,6 +918,19 @@ void initNvFuserPythonBindings(PyObject* module) {
   NVFUSER_PYTHON_BINDING_CAST_OP("cast", castOp)
 #undef NVFUSER_PYTHON_BINDING_CAST_OP
 
+  nvf_ops.def(
+      "squeeze",
+      [](nvfuser::FusionDefinition::Operators& self,
+         nvfuser::Tensor* arg,
```

Comment:
is there no way to get `shape` from `nvfuser::Tensor`?

Reply:
No, there's no shape propagation.

Reply:
So `nvfuser::Tensor` doesn't have a shape? How can it operate then?

Reply:
Currently, we need a concrete shape for `squeeze`, not symbolic.

At the time of building the graph, the only information that is needed for nvFuser to define its tensor object and is available here is the number of dimensions and a vector of booleans to indicate whether a particular dimension is contiguous or not. 

> How can it operate then?

The shape of a tensor comes at kernel execution time.